lib/oaken/seeds.rb

```diff
@@ -62,17 +62,8 @@ class << self
     #   class PaginationTest < ActionDispatch::IntegrationTest
     #     setup { seed "cases/pagination" }
     #   end
-    def seed(*directories)
-      Oaken.lookup_paths.product(directories).each do |path, directory|
-        load_from Pathname(path).join(directory.to_s)
-      end
-    end
-
-    private def load_from(path)
-      @loader = Oaken::Loader.new path
```

Comment:
We don't need this `@loader` instance variable, since we're not using it anymore.